### PR TITLE
Rewrite exchange fund flow to match the new database layout

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary
It happened that the schema of the database measurements also changed, so this PR changes the way sanbase interacts with influxdb

### CODE REVIEW

**Comment by valo** [comment]
I am wondering if we should change this query to return the IN - OUT transaction volume. Is that going to make the thing simpler? IMO, this is the stat we really care about and we won't be showing the IN and OUT volumes separately.

**Comment by IvanIvanoff** [comment]
You mean the difference between `SUM(in_amount)` and `SUM(out_amount)` for every of the `GROUP BY` intervals? This actually will catch and not display movements between team wallets as ETH Spent most of the time. Though there are some edge cases like if the in and out transactions are in different intervals

**Comment by IvanIvanoff** [comment]
@valo done

File: lib/sanbase_web/graphql/resolvers/etherbi_resolver.ex [hunk 1]
```diff
@@ -120,4 +119,36 @@ defmodule SanbaseWeb.Graphql.Resolvers.EtherbiResolver do
...
+        amount: in_amount,
+        transaction_type: "in"
+      },
+      %{
+        datetime: datetime,
+        amount: out_amount,
+        transaction_type: "out"
+      }
+    ]
+  end
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/etherbi_resolver.ex]: I think these 3 functions are not used any more

**Pull request merged**